### ISSUE TITLE
Cannot edit passwords without unhiding it first

### ISSUE THREAD

**Issue by Taomyn** [bitwarden/clients#7889]
### Steps To Reproduce

1. Open extension
2. Open any saved login
3. Click edit
4. Cannot change password when "*********"
5. Click on unhide button
6. Password can be changed

### Expected Result

Behaviour before 2024.2.0 was that the password field was editable even when hidden

### Actual Result

Cannot edit password without unhiding it first

### Additional Context

This was never the case until release 2024.2.0, I first noticed it on Edge browser and then when my Brave browsers started getting the update they all started doing the same.

### Operating System

Windows

### Operating System Version

All

### Web Browser

Microsoft Edge, Brave

### Browser Version

All

### Build Version

2024.2.0

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by Adedamola-Aina** [comment]
Hi @Taomyn 

I attempted to reproduce this issue but was unable to do so. I think what you are describing could have its origin in a misconfiguration or how the extension works. 

We use GitHub issues as a place to track bugs and other development related issues. If your issue persists, please write us back using our “Contact support” form located on our Help Center (https://bitwarden.com/help/).

You can include a link to this issue in the message content and a recording of this behavior. 

The issue here will be closed.

**Comment by Taomyn** [comment]
I know you closed it, but I'm adding this here showing the problem

https://github.com/bitwarden/clients/assets/4365929/25fb0662-36ab-44bb-9535-c385b06b1745

**Comment by uedvt359** [comment]
Hi, @Adedamola-Aina. I think you might have closed this ticket a little bit too soon. We can reproduce this as well. After clicking "edit", the password shows as bullets **and appears editable**, but changes to it will be **silently discarded** when clicking save. 

The issue is either that the placeholder element (which shows the password as bullets) should not be editable, or better yet, when modifying a password in hidden state changes should persist.

reproducable in Win10 with Edge Version 121.0.2277.83 (Official Build) (64-Bit) and BW Plugin Version: 2024.2.0

**Comment by Taomyn** [comment]
@uedvt359 for me "the password shows as bullets and appears editable" is not the case and is completely read-only, it's only editable when I expose the password. No matter how I click on the bullets, it only highlights them but I cannot type anything into the field.

**Comment by uedvt359** [comment]
yes, you're right. I mixed up the old plugin version (2024.1.1), which worked with the new broken version (2024.2.0). i fixed my comment.

**Comment by davideverall** [comment]
I have also been able to reproduce this on version 2024.2.0 on:

- Chrome Version 121.0.6167.161 (Official Build) (64-bit)
- Edge Version 121.0.2277.112 (Official build) (64-bit)

**Comment by Lincest** [comment]
This problem still exists in 2024.2.0

**Comment by RhysMcW** [comment]
this only looks to be an issue in the BW browser add-on, I have this issue in Chrome, Edge and Firefox.
I can edit passwords in the desktop version without having to make the password visible first (Version 2024.2.0)

**Comment by jprusik** [comment]
We'll be reverting this behaviour and will post here with updates as that task progresses.

**Comment by jprusik** [comment]
The PR to remedy is currently in QA tessting

**Comment by jprusik** [comment]
The PR for this has passed QA and has been merged. Expect to see it with the next release 👍

**Comment by user00036** [comment]
> 此 PR 已通过 QA 并已合并。 期待在下一个版本中看到它👍

When will the next version be released?
